SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -49,6 +49,21 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 	void OnShowFailedNotification(const FString& NotificationText);
 	void OnShowTaskStartNotification(const FString& NotificationText);
 
+	FReply OnLaunchDeployment();
```

Comment:
Can some of the new functions be put into private section?

Reply:
Actually they're invoked from another module as well, so must be public, in the future we can consider to create a more general class to put these methods since currently toolbar is a UI class which is not an appropriate place to hold them